plugin/builtin/buildbaron/build_baron.go

```diff
@@ -43,6 +49,20 @@ type bbPluginOptions struct {
 type bbProject struct {
 	TicketCreateProject  string   `mapstructure:"ticket_create_project"`
 	TicketSearchProjects []string `mapstructure:"ticket_search_projects"`
+
+	AlternativeEndpointURL struct {
```

Comment:
Same here- these should be defined outside the bbProject struct though they can be public. Also maybe a pointer